clients/ts/signalr/src/HubConnection.ts

```diff
@@ -39,6 +41,8 @@ export class HubConnection {
      */
     public serverTimeoutInMilliseconds: number;
 
+    public pingIntervalInMilliseconds: number;
```

Comment:
Can you add JSDoc comments to this, similar to the XML docs for C#? Use the comments above `serverTimeoutInMilliseconds` as a guide. The first sentence is a brief summary, followed by a blank line, followed by a more detailed explanation.